app/models/report_type.rb

```diff
@@ -2,26 +2,57 @@
 #
 # Table name: report_types
 #
-#  id            :integer          not null, primary key
-#  abbr          :string           not null
-#  full_name     :string           not null
-#  class_name    :string           not null
-#  custom_report :boolean          default(FALSE)
-#  seq_id        :integer          not null
+#  id                       :integer          not null, primary key
+#  abbr                     :string           not null
+#  full_name                :string           not null
+#  class_name               :string           not null
+#  custom_report            :boolean          default(FALSE)
+#  seq_id                   :integer          not null
+#  foi                      :boolean          default(FALSE)
+#  sar                      :boolean          default(FALSE)
+#  standard_report          :boolean          default(FALSE), not null
+#  default_reporting_period :string           default("year-to-date")
 #
 
 class ReportType < ApplicationRecord
+
+  VALID_DEFAULT_REPORTING_PERIODS = %w{
+                                        year_to_date
+                                        quarter_to_date
+                                        last_quarter
+                                        last_month
+                                      }
+
```

Comment:
It'd be nice to have this as an `enum`. It appropriately descriptive and would give us scopes and validation for free.